_includes/home-banner.html

```diff
@@ -9,12 +9,20 @@ <h1>
       </h1>
       <p>Open-source design for bitcoin products.</p>
       <div class="home-banner-buttons-wrapper">
-        <a id="slack-link-button" href="https://bitcoindesign.slack.com/" target="blank" rel="me"> Join us on Slack</a>
+        <a id="slack-link-button" href="https://bitcoindesign.slack.com/" target="blank" rel="me">Join us on Slack</a>
```

Comment:
Invite link should be https://join.slack.com/t/bitcoindesign/shared_invite/zt-gytq2snl-4TEWJOTKrXRCB4YLBoDunA

Reply:
I've heard these expire after some time?

Reply:
This one doesn't have an expiration, but in fact @moneyball just made me aware of http://bitcoindesigners.org in PR https://github.com/BitcoinDesign/Guide/pull/28#issuecomment-696842741 which also doesn't expire but is also easier to remember and type out.